src/java/org/apache/cassandra/net/MessagingService.java

```diff
@@ -1664,4 +1676,113 @@ public static boolean isEncryptedConnection(InetAddressAndPort address)
         }
         return true;
     }
+
+    public void blockForPeers()
+    {
+        // TODO make these yaml props?
```

Comment:
well, I wanted to get more input before shovelling more stuff into the yaml :) If we want users to actually use this in a 'standard' way, then they probably should be in the yaml

Reply:
This is default on so no I don't think we need to add it to the YAML just clean up the TODO.

Reply:
If we choose to not put these params (`alivePercent` and `aliveTimeoutSecs`) into the yaml, should they be documented and added to `conf/jvm.options`?

Reply:
If we are going that route it seems like well then it should just be in the YAML. I would say jvm.options should be for things we don't control because they aren't directly part of Cassandra.

Another option is as a YAML option but undocumented if you want to not clutter the YAML, but then operators get annoyed and confused and ask why it's not a yaml option and not documented.

I never really know how to balance the tension of not putting out thousands of config options into a file with not having undocumented but important knobs. I think this one doesn't need to be in the documented category yet because setting it is really as an escape hatch not something people should have to worry about. This is just some feature that should silently work and people don't have to think about. Then later if we find out we are wrong we can add the option and document. This stems the tide of config options people need to think about which has value.

Reply:
ok, sgtm. I will add the params to `Config`, but not add to the yaml.